src/components/tx/TxData.js

```diff
@@ -535,7 +559,9 @@ class TxData extends React.Component {
               {conflictNotTwin.length > 0 && (
                 <div className="mb-0">
                   <span>Transactions double spending the same outputs as this transaction: </span>
-                  {renderListWithLinks(conflictNotTwin, true)}
+                  {this.props.newUiEnabled
+                    ? renderNewUiListWithLinks(conflictNotTwin)
+                    : renderListWithLinks(conflictNotTwin, true)}
```

Comment:
Since this is used many times on this file, maybe the better way to do this would be to move the conditional to the `renderListWithLinks` so only the method changes and the rest of the file remains the same.

Reply:
I’ve applied your suggestion and refactored the code accordingly. Now, instead of calling renderNewUiListWithLinks multiple times, we only call renderListWithLinks and handle the validation of what it returns within the method itself. This should make the code cleaner and more maintainable. Let me know if you need any further adjustments!